src/Page/Page.php

```diff
@@ -92,7 +92,7 @@ protected function verifyUrl(array $urlParameters = []): void
         }
     }
 
-    protected function getParameter(string $name): NodeElement
+    protected function getParameter(string $name): ?string
```

Comment:
One thing, should it be `null` allowed indeed?

Reply:
So far i didn't need to get any other parameter than page url, which of course should't be null, but by default there are few other parameters that could (and in my case they are) be null like for example files path.